### ISSUE TITLE
Get errors

### ISSUE THREAD

**Issue by Webdude2023** [levinunnink/html-form-to-google-sheet#24]
**Describe the bug**
A clear and concise description of what the bug is:

{"result":"error","error":{}}

What am I doing wrong? I have followed these steps:
https://github.com/levinunnink/html-form-to-google-sheet

I have tried it from localhost, from html-file, from php-file, online from a Wordpress website. Nothing works. The same problem.

**Comment by Webdude2023** [comment]
> 

No, I have tried to set it up many times, and used different browsers, but nothing works.

**Comment by Webdude2023** [comment]
> I get the same error also, do you have found solution @Webdude2023 please ?

No, I have tried to set it up many times, and used different browsers, but nothing works.

**Comment by salimbenfarhat** [comment]
@Webdude2023 it's works for me now !
some tips : 
- on your gsheet make sur you sheet is called 'Sheet1'. Put all your fields title in lowercase without space and at the first line directly.
- After use this code : 
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <title>Test Form Gsheet</title>
  </head>
  <body>
    <h1>Test Form Gsheet</h1>
    <form name="submit-to-google-sheet">
      <input name="fullname" type="text" placeholder="Full Name" required>
      <input name="email" type="email" placeholder="Email" required>
      <button type="submit">Envoyer</button>
    </form>
   
    <script>
      const scriptURL = 'YOUR SCRIPT URL finishing by exec'
      const form = document.forms['submit-to-google-sheet']
    
      form.addEventListener('submit', e => {
        e.preventDefault()
        fetch(scriptURL, { method: 'POST', body: new FormData(form)})
          .then(response => console.log('Success!', response))
          .catch(error => console.error('Error!', error.message))
      })
    </script>
  </body>
</html>

```

**Comment by salimbenfarhat** [comment]
> @Webdude2023 it's works for me now ! some tips :
> 
> * on your gsheet make sur you sheet is called 'Sheet1'. Put all your fields title in lowercase without space and at the first line directly.
> * After use this code :
> 
> ```
> <!DOCTYPE html>
> <html>
>   <head>
>     <meta charset="utf-8" />
>     <title>Test Form Gsheet</title>
>   </head>
>   <body>
>     <h1>Test Form Gsheet</h1>
>     <form name="submit-to-google-sheet">
>       <input name="fullname" type="text" placeholder="Full Name" required>
>       <input name="email" type="email" placeholder="Email" required>
>       <button type="submit">Envoyer</button>
>     </form>
>    
>     <script>
>       const scriptURL = 'YOUR SCRIPT URL finishing by exec'
>       const form = document.forms['submit-to-google-sheet']
>     
>       form.addEventListener('submit', e => {
>         e.preventDefault()
>         fetch(scriptURL, { method: 'POST', body: new FormData(form)})
>           .then(response => console.log('Success!', response))
>           .catch(error => console.error('Error!', error.message))
>       })
>     </script>
>   </body>
> </html>
> ```

don't forget also to add 'timestamp' column at the left before the other fields
